.github/workflows/pull_request.yml

```diff
@@ -275,3 +275,54 @@ jobs:
                   git add . || echo "No updated snapshots, nothing to add!"
                   git commit -m "chore: update cypress snapshots [ci skip cypress]" --no-verify || echo "No updated snapshots, nothing to commit!"
                   git push
+
+    preview:
+        needs: [build]
+        if: needs.build.outputs.visual == 'true' && !contains(github.event.pull_request.sender, 'fremtind-bot')
+        runs-on: ubuntu-latest
+        permissions:
+            actions: write
+            contents: write
+            issues: write
+        env:
+            PREVIEW_PATH: preview/${{github.event.number}}
```

Comment:
Kan man bruke `${{github.ref_name}}` (gir branch-navn) her for å slippe en ny mappe for hvert push? Eller er det nødvendig med ny mappe for at det skal fungere?

Reply:
Det er pull request-nummer, så det blir samme mappa hver push